manual/idempotence/README.md

```diff
@@ -0,0 +1,132 @@
+## Query idempotence
+
+A query is *idempotent* if it can be applied multiple times without changing the result of the initial application. For
```

Comment:
Should we add emphasis here that you should make sure that you use client timestamp generators when using idempotent queries?  Although that is default now so maybe not as important to note.